### DISCUSSION TITLE
Capitalize the E of email custom field and Proton Pass `alias` type handling

### DISCUSSION THREAD

**Discussion by tumatanquang** [bitwarden/clients#11934] [Password Manager]
### Code Contribution Proposal

As discussed in this topic: [Username data field is skipped when importing data from Proton Pass](https://community.bitwarden.com/t/username-data-field-is-skipped-when-importing-data-from-proton-pass/75498/6)

## Capitalize the E of email custom field:

This is pretty simple, change the code of [line 143 in the protonpass-json-importer.ts](https://github.com/bitwarden/clients/blob/main/libs/importer/src/importers/protonpass/protonpass-json-importer.ts#L143) file from:

```
this.processKvp(cipher, "email", loginContent.itemEmail);
```

To:

```
this.processKvp(cipher, "Email", loginContent.itemEmail);
```

This will help to recognize this as a title/value, avoiding being mistaken for a `type` or something similar.

## Proton Pass `alias` type handling:

Currently, the Proton Pass `alias` type is ignored when importing, which causes missing data.
As discussed in the above topic, we can save this `type` as a `note`.
The JSON structure and corresponding handling are [available here](https://community.bitwarden.com/t/username-data-field-is-skipped-when-importing-data-from-proton-pass/75498/4).
I hope someone will add code to handle this.

**Comment by dwbit** [comment]
Hi there, feature requests can be posted here: https://community.bitwarden.com/c/feature-requests/pm-feature-requests/55 If you're looking to contribute to the codebase by writing code, please provide additional information.
